tests/framework/installer/ceph_installer.go

```diff
@@ -267,8 +267,8 @@ func (h *CephInstaller) CreateCephCluster() error {
 
 func (h *CephInstaller) waitForCluster() error {
 	monWaitLabel := "app=rook-ceph-mon,mon_daemon=true"
-	if h.Manifests.Settings().RookVersion == Version1_14 {
-		// TODO: Remove this when upgrade test is from v1.15 since v1.14 does not have the mon_daemon label
+	if h.Manifests.Settings().RookVersion == Version1_15 {
+		// TODO: Remove this when upgrade test is from v1.15.7 since prior releases do not have the mon_daemon label
```

Comment:
```suggestion
```

time to remove this?

Reply:
That new label actually wasn't added until #15046 was backported a few days ago, so that fix will be included in v1.15.7. So we need to keep this behavior for now.